docs/operations.md

```diff
@@ -186,12 +183,12 @@ alerting will be documented here.
 See [Wormhole.json](../dashboards/Wormhole.json) for an example Grafana dashboard.
 
 **NOTE:** Parsing the log output for monitoring is NOT recommended. Log output is meant for human consumption and is
-not considered a stable API. Log messages may be added, modified or removed without notice. Use the metrics :-)
+not considered a stable API. Log messages may be added, modified, or removed without notice. Use the metrics :-)
```

Comment:
Please don’t switch everything to an Oxford comma. This is needless churn in the docs with zero actual improvements. Fixing actual typos or grammatical errors is fine.

Reply:
Hey the only goal was to change the Terra node links but Grammarly suggested some edits thats why did it thought might be useful, sorry 😅